src/test/java/org/trebol/jpa/services/conversion/UserRolesConverterJpaServiceImplTest.java

```diff
@@ -0,0 +1,71 @@
+package org.trebol.jpa.services.conversion;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.core.convert.ConversionService;
+import org.trebol.exceptions.BadInputException;
+import org.trebol.jpa.entities.UserRole;
+import org.trebol.pojo.UserRolePojo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class UserRolesConverterJpaServiceImplTest {
+
+    @InjectMocks
+    private UserRolesConverterJpaServiceImpl sut;
+
+    @Mock
+    private ConversionService conversionService;
+
+    private UserRole userRole;
+    private UserRolePojo userRolePojo ;
+
+    @BeforeEach
+    void beforeEach() {
+        userRole = new UserRole();
+        userRole.setName("ANY");
+        userRole.setId(1L);
+
+        userRolePojo = new UserRolePojo();
+        userRolePojo.setId(1L);
+        userRolePojo.setName("ANY");
+    }
+
+    @AfterEach
+    void afterEach() {
+        userRole = null;
+        userRolePojo = null;
+    }
+
+    @Test
+    void testApplyChangesToExistingEntity() throws BadInputException {
+        UserRole actual = sut.applyChangesToExistingEntity(userRolePojo, userRole);
+        assertEquals(1L, actual.getId());
```

Comment:
This test doesn't do much since both objects are pretty much sharing the same data.